manifests/o/OrpheusTechnologyLtd/ProWritingAidEverywhere/3.2.2563.3/OrpheusTechnologyLtd.ProWritingAidEverywhere.locale.en-US.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OrpheusTechnologyLtd.ProWritingAidEverywhere
+PackageVersion: 3.2.2563.3
+PackageLocale: en-US
+Publisher: Orpheus Technology Ltd
+PrivacyUrl: https://prowritingaid.com/en/Home/Privacy
+PackageName: ProWritingAid Everywhere
+PackageURL: https://prowritingaid.com/everywhere-download-windows

```

Comment:
```suggestion
PackageUrl: https://prowritingaid.com/everywhere-download-windows
```

Reply:
Apologies, I had incorrect capitalization. I've tested this suggestion to be sure it will work.